main.tf

```diff
@@ -150,6 +150,27 @@ resource "confluent_api_key" "service_account_api_keys" {
   }
 }
 
+resource "confluent_api_key" "new_service_account_api_keys" {
+  for_each     = toset(local.service_accounts)
+  display_name = "${local.name}-${each.value}-${random_pet.pet.id}-api-key"
```

Comment:
what is random_pet.pet.id? not familiar with this

Reply:
ah, interesting https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet